tests/Argument/ArgumentResolverTest.php

```diff
@@ -49,7 +50,7 @@ public function testResolverResolvesResolvesRawArguments()
         $container = $this->getMockBuilder(Container::class)->getMock();
 
         $container->expects($this->at(0))->method('has')->with($this->equalTo('alias1'))->willReturn(true);
-        $container->expects($this->at(1))->method('get')->with($this->equalTo('alias1'))->willReturn(new RawArgument('value1'));
+        $container->expects($this->at(1))->method('get')->with($this->equalTo('alias1'))->willReturn('value1');
```

Comment:
i don't see when container::get would ever return a RawArgument object? So I simplified this. I can undo it if you want.

Reply:
yeah, not sure what I was thinking here haha, thanks